repos/rhel9-internal.repo

```diff
@@ -1,5 +1,5 @@
 [rhel-9.0-baseos]
 name=rhel9 baseos
-baseurl=https://rhsm-pulp.corp.redhat.com/content/eus/rhel9/9.0/$basearch/baseos/os/
+baseurl=https://rhsm-pulp.corp.redhat.com/content/eus/rhel9/9.1/$basearch/baseos/os/
```

Comment:
@cfergeau Right now https://rhsm-pulp.corp.redhat.com/content/eus/rhel9/ doesn't have 9.1 directory, it is going to break the link.

Reply:
Ah thanks! I was relying on our CI to tell us if the link exists :)

Reply:
our CI doesn't run arm bits so this might not even run :(